README.md

```diff
@@ -61,8 +61,8 @@ We want to make contributing to this project as easy and transparent as possible
 
 As of April 5, 2017, Parse, LLC has transferred this code to the parse-community organization, and will no longer be contributing to or distributing this code.
 
- [parse.com]: https://parseplatform.github.io/
- [guide]: http://parseplatform.github.io/docs/android/guide/
+ [parse.com]: http://docs.parseplatform.org/
```

Comment:
maybe rename [parse.com] to [parseplatform.org] (here: https://github.com/parse-community/Parse-SDK-Android/pull/634/files#diff-04c6e90faac2675aa89e2176d2eec7d8R9) and use `[parseplatform.org](http://parseplatform.org)`